servers/rendering/renderer_rd/shader_rd.cpp

```diff
@@ -616,7 +616,7 @@ void ShaderRD::version_set_code(RID p_version, const HashMap<String, String> &p_
 	ERR_FAIL_COND(is_compute);
 
 	Version *version = version_owner.get_or_null(p_version);
-	ERR_FAIL_NULL(version);
+	ERR_FAIL_NULL_MSG(version, "Can't use a shader that previously failed compilation.");
```

Comment:
As per my last review, this isn't a compile error. Failing to compile is just one reason why this case may be hit. The message is misleading

Reply:
I've reworded all 3 messages somewhat. The concept of "shader version" isn't something that is obvious to most users though (unless you look at the native shader source inspector), so I'm not sure how we could clarify it.

Reply:
Ideally this is an error that users would never see. It's meant to be totally internal